Library/Homebrew/dev-cmd/update-test.rb

```diff
@@ -81,7 +81,7 @@ def update_test
       safe_system "git", "reset", "--hard", start_commit
 
       # update ENV["PATH"]
-      ENV["PATH"] = "#{curdir}/bin:/usr/local/bin:/usr/bin:/bin"
+      ENV["PATH"] = "#{curdir}/bin:#{ENV["PATH"]}"
```

Comment:
Use the `PATH` class here.

Reply:
I have no objections to that happening in a follow-up PR but this is biting users and needs merged ASAP and this refactoring isn't a priority for that, sorry.